examples/undocumented/libshogun/parameter_modsel_parameters.cpp

```diff
@@ -23,8 +23,6 @@ void print_message(FILE* target, const char* str)
 
 void print_modsel_parameters(CSGObject* object)
```

Comment:
we can just drop this function

Reply:
actually, we can just drop the whole cpp example ... to be replaced with a meta example

Actually 
@gf712 could you try to write a meta example for `base_api` where you show how to extract model selection parameters at some point? mayber after the monkey business is done